library/src/main/java/org/xmtp/android/library/Client.kt

```diff
@@ -291,30 +291,32 @@ class Client() {
 
     // This is a V3 only feature
     suspend fun createOrBuild(
-        account: SigningKey,
-        options: ClientOptions,
+        address: String,
+        account: SigningKey? = null,
+        options: ClientOptions? = null,
```

Comment:
Since this is createOrBuild we only need the signingKey for the create action and the address for the build action. So lets make this nullable.